javascript/ql/test/query-tests/Security/CWE-116/IncompleteSanitization/tst.js

```diff
@@ -126,6 +126,21 @@ function good11(s) {
   return s.replace("%d", "42");
 }
 
+function good12(s) {
+	s.replace('[', '').replace(']', ''); // OK
+	s.replace('(', '').replace(')', ''); // OK
+	s.replace('{', '').replace('}', ''); // OK
+
+	s.replace('[', '\\[').replace(']', '\\]'); // probably OK, but still flagged
+	s.replace('(', '\\[').replace(')', '\\)'); // OK
+	s.replace('{', '\\{').replace('}', '\\}'); // probably OK, but still flagged
+
+	s = s.replace('[', ''); // OK, but still flagged
+	s = s.replace(']', ''); // OK, but still flagged
```

Comment:
Presumably you could handle this by using local data flow in `isDelimiterUnwrapper`?